ui/language_utils.js

```diff
@@ -82,6 +87,21 @@ shaka.ui.LanguageUtils = class {
       return name;
     };
 
+    const getAudioCodecName = (audioCodec) => {
```

Comment:
I've been thinking about it, and I think there should be no configuration and we should do the same thing we do in https://github.com/shaka-project/shaka-player/blob/main/ui/resolution_selection.js#L259

For the rest LGTM.

Reply:
Done!